src/components/MultiSelect/MultiSelect.test.tsx

```diff
@@ -143,153 +110,134 @@ describe.skip("MultiSelect Dialog", () => {
   it("should initially render with provided id", () => {
     const { container } = render(
       <MultiSelect
-        id="multiselect-dialog-test-id"
-        labelText="MultiSelect Label"
-        type="dialog"
+        id="multiselect-test-id"
+        helperText="multiselect-helper-text"
+        buttonText="multiselect-button-text"
+        isDefaultOpen={false}
+        isSearchable={false}
+        isBlockElement={false}
+        defaultItemsVisible={defaultItemsVisible}
         items={items}
         selectedItems={selectedTestItems}
         onChange={() => null}
         onClear={() => null}
-        onApply={() => null}
       />
     );
-    expect(
-      container.querySelector("#multiselect-dialog-test-id")
-    ).toBeInTheDocument();
+    expect(container.querySelector("#multiselect-test-id")).toBeInTheDocument();
   });
 
   it("should initially render with a given label", () => {
     render(
       <MultiSelect
-        id="multiselect-dialog-test-id"
-        labelText="MultiSelect Test Label"
-        type="dialog"
+        id="multiselect-test-id"
+        helperText="multiselect-helper-text"
+        buttonText="multiselect-button-text"
+        isDefaultOpen={false}
+        isSearchable={false}
+        isBlockElement={false}
+        defaultItemsVisible={defaultItemsVisible}
         items={items}
         selectedItems={selectedTestItems}
         onChange={() => null}
         onClear={() => null}
-        onApply={() => null}
       />
     );
-
-    expect(
-      screen.getByRole("button", {
-        name: /multiselect test label/i,
-      })
-    ).toBeInTheDocument();
+    expect(screen.getByRole("button")).toBeInTheDocument();
```

Comment:
This is fine because it checks for the button, but the test description says it renders a given label and the label is not being tested here.